internal/command/deploy/plan.go

```diff
@@ -493,11 +494,16 @@ func (md *machineDeployment) updateMachineWChecks(ctx context.Context, oldMachin
 	if !healthcheckResult.regularChecksPassed {
 		sl.LogStatus(statuslogger.StatusRunning, fmt.Sprintf("Checking health of machine %s", machine.ID))
 		err = lm.WaitForHealthchecksToPass(ctx, md.waitTimeout)
-		if err != nil {
+		switch {
+		case errors.Is(err, context.DeadlineExceeded):
```

Comment:
didn't mean to commit this, but this is a small fix to retry health checks correctly.